shared/message/platform/darwin/DarwinSensors.proto

```diff
@@ -85,27 +162,113 @@ message DarwinSensors {
         uint32 errorFlags = 7;
     }
 
+    /**
+     * Pair of FSR messages - one for each foot.
+     */
     message FSRs {
-        FSR left  = 1;
+
+        /**
+         * Left foot set of FSRs, with centre coordinates and error flags.
+         */
+        FSR left = 1;
+
+        /**
+         * Right foot set of FSRs, with centre coordinates and error flags.
+         */
         FSR right = 2;
     }
 
+
+    /**
+     * Full set of information to use a given servo.
+     * Ref: https://emanual.robotis.com/docs/en/dxl/mx/mx-64/ ; https://emanual.robotis.com/docs/en/dxl/mx/mx-106/
+     */
     message Servo {
-        uint32 errorFlags      = 1;
-        bool   torqueEnabled   = 2;
-        float  pGain           = 3;
-        float  iGain           = 4;
-        float  dGain           = 5;
-        float  goalPosition    = 6;
-        float  movingSpeed     = 7;
-        float  torque          = 8;
-        float  presentPosition = 9;
-        float  presentSpeed    = 10;
-        float  load            = 11;
-        float  voltage         = 12;
-        float  temperature     = 13;
+
+        /**
+         * Set of things which can or have gone wrong.
+         * https://emanual.robotis.com/docs/en/dxl/mx/mx-106/#alarm-led17-shutdown18
+         */
+        uint32 errorFlags = 1;
+
+        /**
+         * Whether the torque is on or off.
+         */
+        bool torqueEnabled = 2;
+
+        /**
+         * Proportional gain. In proportion to the servo's position error. Gain values are in range 0~254.
+         * K_p : p_gain / 8
+         */
+        float pGain = 3;
+
+        /**
+         * Integral gain. Increases in relation to the error, and how long the error has persisted. Gain values are in
+         * range 0~254. K_i : i_gain * 1000 / 2048
+         */
+        float iGain = 4;
+
+
+        /**
+         * Derivative gain. Does not consider error, just dampens forces to reduce derivatives. Gain values are in range
+         * 0~254. K_d : d_gain * 4 / 1000
+         */
+        float dGain = 5;
+
+        /**
+         * It is a position value of destination. 0 to 4,095 (0xFFF) is available. The unit is 0.088 [°].
+         * (FROM https://emanual.robotis.com/docs/en/dxl/mx/mx-64/#specifications):
+         * If Goal Position is out of the range, Angle Limit Error Bit (Bit1) of Status Packet is returned as ‘1’ and
+         * Alarm is triggered as set in Alarm LED/Shutdown.
+         */
+        float goalPosition = 6;
+
+        float movingSpeed = 7;
```

Comment:
What is moving speed???

Reply:
Moving speed is what dynamixel calls servo velocity